RELEASES.md

```diff
@@ -0,0 +1,248 @@
+
+	```console
+	$ git cherry-pick -xsS <commit>
+	```
+4. Push the branch and open up a PR against the _release branch_:
+
+	```
+	$ git push -u stevvooe my-backport-branch
+	```
+
+   Make sure to replace `stevvooe` with whatever fork you are using to open
+   the PR. When you open the PR, make sure to switch `master` with whatever
+   release branch you are targeting with the fix.
+
+If there is no existing fix in master, you should first fix the bug in master,
+or ask us a maintainer or contributor to do it via an issue. Once that PR is
+completed, open a PR using the process above.
+
+Only when the bug is not seen in master and must be made for the specific
+release branch should you open a PR with new code.
+
+## Public API Stability
+
+The following table provides an overview the components covered by containerd
+versions:
+
+
+| Component  | Status   | Stablized Version | Links         |
+|------------|----------|-------------------|---------------|
+| GRPC API   | Alpha    | 1.0               | [api/](api) |
+| Go API     | Unstable | 1.1 tentative     | [godoc](https://godoc.org/github.com/containerd/containerd) |
+| `ctr` tool | Unstable | Out of scope      | -             |
+
+From the version stated in the above table, that component must adhere to the
+stability constraints expected in release versions.
+
+Unless explicitly stated here, components that are called out as unstable or
+not covered may change in a future minor version. Breaking changes to
+"unstable" components will be avoided in patch versions.
+
+### GRPC API
+
+The primary product of containerd is the GRPC API. As of the 1.0.0 release, the
+GRPC API will not have any backwards incompatible changes without a _major_
+version jump.
+
+To ensure compatibility, we have collected the entire GRPC API symbol set into
+a single file. At each minor release of containerd, we will move the current
+`next.pb.txt` file to a file named for the minor version, such as `1.0.pb.txt`,
+enumerating the support services and messages. See [api/](api) for details.
+
+Note that new services may be added in _minor_ releases.
+
+#### Error Codes
+
+Error codes will not change in a patch release, unless a missing error code
+causes a blocking bug. Error codes of type "unknown" may change to more
+specific types in the future. Any error code that is not "unknown" that is
+currently returned by a service will not change without a _major_ release or a
+new version of the service.
+
+If you find that an error code that is required by your application is not
+well-documented in the protobuf service description or tested explicitly,
+please file and issue and we will clarify.
+
+#### Field Microformats
+
+Unless explicitly stated, the formats of certain fields may not be covered by
+this guarantee and should be treated opaquely. For example, don't rely on the
+format details of a URL field unless we explicitly say that the field will
+follow that format.
+
+### Go API
+
+The Go API, documented in
```

Comment:
I guess this section is mainly about the client library.
Does it apply for plugins as well?